governance-exercises/poll-ex3-responses.md

```diff
@@ -0,0 +1,28 @@
+## Poll Exercise 3 Responses
+
+Here are the actions I would take:
+
+1. First, I would validate the text of the standing order to ensure it aligns with Maker's principles, even if some ambiguities exist. I would also take proactive steps to resolve the ambiguity in the text of the standing order by reaching out to Emilia privately to discuss further on Friday
+2. Next, I would recognize MagicRWA's proposal with a forum response and note that it will be placed in a weekly poll for community consideration, given the significance of the changes and their implications.
```

Comment:
There are several reasons to delay / refuse a poll here. 
* First reason is that the timeline is unreasonably short for a non-emergency change, and this is a non-emergency change. Faraz's approval also comes in on Monday, the day of the poll, again too short notice to be actioned outside of an emergency.
* This breaks the existing ALM transfer system (which does protect Maker in emergencies) without replacing it. Faraz failing to comment on this meaningfully as stability facilitator should be extremely concerning.
* The inclusion of the forum poll here is somewhat misleading, it's also done extremely badly, polling on multiple values in pre-defined combinations. That they've done this presumably without consulting you about it is fairly bad. It introduces lots of risks around that poll outcome, and the eventual wording of the order (if it doesn't match.) It's also just the sort of question that risk experts are better at answering than governance as a whole. 
* It's also unclear exactly what Faraz is referring to in their forum post: "We approve this standing order changes for a poll today." This is likely just a typo, but where is the typo? There is a small possibility that Faraz believes that the vote will include two standing order changes: this -> these. This should be ruled out before polling. 

Ultimately, there is no reason to invalidate the existing standing order now with this change, especially when the replacement is meant to be ready shortly. Better to wait until both of the new orders are ready, and act then.

Reply:
One more, the phrasing: "This poll should take place on Monday" is a red flag. Its not in Emilia's authority to determine when a poll should take place. It's a problem if they don't know that. Given the desire for a compressed timeline, they should be asking politely if the Governance facilitators can make an exception to do the poll on Monday.